autoload/codefmt.vim

```diff
@@ -26,6 +26,7 @@
 " system.
 "
 " The current list of defaults by filetype is:
+"   * BUILD files (Bazel): buildifier
```

Comment:
The rest of these are indexed by filetype, so this might be better as:

```VimL
"   * bzl (Bazel BUILD files): buildifier
```

Though as David mentions, I'm not sure it's necessarily worth duplicating this list all over the place. (For example, this version is missing mention of js-beautify.)